api/envoy/service/status/v3/csds.proto

```diff
@@ -88,7 +90,13 @@ message PerXdsConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.PerXdsConfig";
 
+  // Config status is generated by management servers.
   ConfigStatus status = 1;
+
+  // Client side config status is populated by xDS clients. Only one of the
```

Comment:
Instead of "Only one of the config status should present in one PerXdsConfig message", let's just say "Will not be present if the CSDS server is an xDS server."

Reply:
Done. That's better.